app/notify_admin.go

```diff
@@ -217,7 +258,25 @@ func (a *App) FinishSendAdminNotifyPost(trial bool, now int64) {
 		mlog.Error("Unable to finish send admin notify post job", mlog.Err(err))
 	}
 
-	// all the notifications are now sent in a post and can safely be removed
+	// all the requested features notifications are now sent in a post and can safely be removed except
+	// the plugin notify admin. we keep it as we do not want the same user to send the notification for the same plugin
+	// from the work template feature. We update the NotifyAdmin sentAt to keep track of it.
+	if len(pluginBasedData) > 0 {
+		for pluginId := range pluginBasedData {
+			notifications := pluginBasedData[pluginId]
+			for _, notification := range notifications {
+				requiredFeature := notification.RequiredFeature
+				if strings.HasPrefix(string(requiredFeature), string(model.WorkTemplates)) {
+					requiredPlan := notification.RequiredPlan
+					userId := notification.UserId
+					if err := a.Srv().Store().NotifyAdmin().Update(userId, requiredPlan, requiredFeature, now); err!=nil {
+						mlog.Error("Unable to update SentAt for work template feature", mlog.Err(err))
```

Comment:
Server platform team wants to move from using `mlog` to `a.Log()`. I'd suggest using that here, even though `mlog` is used elsewhere in this file.